assets/src/dashboard/components/dropdown/index.js

```diff
@@ -39,24 +41,29 @@ const Label = styled.label`
 
 export const InnerDropdown = styled.button`
   align-items: center;
-  background-color: ${({ theme, transparent }) =>
-    transparent ? 'transparent' : theme.colors.gray25};
-  border-radius: 4px;
-  border: 1px solid transparent;
+  background-color: ${({ theme, type, isOpen }) =>
+    theme.dropdown[type][isOpen ? 'activeBackground' : 'background']};
+  border-radius: ${({ theme, type }) => theme.dropdown[type].borderRadius};
+  border: ${({ theme, type }) => theme.dropdown[type].border};
   color: ${({ theme }) => theme.colors.gray600};
   cursor: ${({ disabled }) => (disabled ? 'inherit' : 'pointer')};
   display: flex;
   font-family: ${({ theme }) => theme.fonts.dropdown.family};
   font-size: ${({ theme }) => theme.fonts.dropdown.size};
   font-weight: ${({ theme }) => theme.fonts.dropdown.weight};
   height: 40px;
```

Comment:
Super nit, it looks like the pill drop-downs are supposed to be 48px tall:
![image](https://user-images.githubusercontent.com/40646372/77473862-53d1b200-6dd3-11ea-8e55-6712d28a3cf2.png)